source/infrastructure.rst

```diff
@@ -0,0 +1,75 @@
+Infrastructure
+==============
+
+The PyCon UK infrastructure is comprised of the following services:
+
+**Website**
+
+* GitHub for code
+* GitHub for hosting
+* Travis-CI for code testing
+* DNS ?
+* Domain (123-reg)
+
+
+**Ticketing**
+
+* ti.to
+
+
+**Organisation**
+
+* GitHub for Issues
+* GitHub for docs code
+* RTD for docs hosting
+
+
+**SMS App**
+
+* GitHub for code
+* Twilio for SMS delivery
+
+**Voting App**
+
+* Heroku for hosting
+* GitHub for code
+* Opbeat for Error tracking
+
+
+This boils down to the following Vendors:
+
+* 123-reg
+* GitHub
+* Heroku
+* Opbeat
+* ReadTheDocs
+* Travis-CI
+* Twilio
+
+
+Pricing
+-------
+
+Ti.to
+`````
+Based on their `pricing page <https://ti.to/pricing>`_ and using the dummy values of 300 tickets at £200p/ticket (note: not actual values) we would be looking at paying between £3176.46 and £3600.
```

Comment:
I did some maths in the budget, and estimated £700 Stripe fees, £670 ti.to fees (ti.to Pro, with 1:1 EUR:GBP)

Can I grab you sometime to compare our parameters etc.? (Mine's in "Humans In" in't budget spreadsheet)